docs/internals/automation.md

```diff
@@ -13,3 +13,8 @@ There are some tasks that are done automatically when working on Yii:
   Run `./build/build php-doc/fix` to fix them.
   Check the changes before you commit them as there may be unwanted changes because the command is not perfect.
   You may use `git add -p` to review the changes.
+
+- Updating the Mime Type Magic file (`framework/helpers/mimeTypes.php`) from the Apache HTTPd repository.
+  Run `./build/build mime-type` to update the file.
+
+All of the above commands are included in the [release process](). They may be run between releases but it is not necessary.
```

Comment:
I have completed the release command while doing the 2.0.8 release, will merge this soon.